examples/nirfmxnr/acp-txp-modacc-single-carrier.py

```diff
@@ -0,0 +1,467 @@
+        )
+    )
+
+    client.CfgIQPowerEdgeTrigger(
+        nirfmxnr_types.CfgIQPowerEdgeTriggerRequest(
+            instrument=instr,
+            selector_string="",
+            iq_power_edge_source="0",
+            iq_power_edge_slope=nirfmxnr_types.IQ_POWER_EDGE_TRIGGER_SLOPE_RISING,
+            iq_power_edge_level=-20.0,
+            trigger_delay=0.0,
+            trigger_min_quiet_time_mode=nirfmxnr_types.TRIGGER_MINIMUM_QUIET_TIME_MODE_AUTO,
+            trigger_min_quiet_time_duration=8.0e-6,
+            iq_power_edge_level_type=nirfmxnr_types.IQ_POWER_EDGE_TRIGGER_LEVEL_TYPE_RELATIVE,
+            enable_trigger=False,
+        )
+    )
+
+    client.SetAttributeI32(
+        nirfmxnr_types.SetAttributeI32Request(
+            instrument=instr,
+            selector_string="",
+            attribute_id=nirfmxnr_types.NIRFMXNR_ATTRIBUTE_LINK_DIRECTION,
+            attr_val_raw=nirfmxnr_types.NIRFMXNR_INT32_LINK_DIRECTION_UPLINK,
+        )
+    )
+
+    client.SetAttributeI32(
+        nirfmxnr_types.SetAttributeI32Request(
+            instrument=instr,
+            selector_string="",
+            attribute_id=nirfmxnr_types.NIRFMXNR_ATTRIBUTE_FREQUENCY_RANGE,
+            attr_val_raw=nirfmxnr_types.NIRFMXNR_INT32_FREQUENCY_RANGE_RANGE1,
+        )
+    )
+
+    client.SetAttributeI32(
+        nirfmxnr_types.SetAttributeI32Request(
+            instrument=instr,
+            selector_string="",
+            attribute_id=nirfmxnr_types.NIRFMXNR_ATTRIBUTE_AUTO_RESOURCE_BLOCK_DETECTION_ENABLED,
+            attr_val_raw=nirfmxnr_types.NIRFMXNR_INT32_AUTO_RESOURCE_BLOCK_DETECTION_ENABLED_TRUE,
+        )
+    )
+
+    client.SetAttributeF64(
+        nirfmxnr_types.SetAttributeF64Request(
+            instrument=instr,
+            selector_string="",
+            attribute_id=nirfmxnr_types.NIRFMXNR_ATTRIBUTE_COMPONENT_CARRIER_BANDWIDTH,
+            attr_val=20e6,
+        )
+    )
+
+    client.SetAttributeF64(
+        nirfmxnr_types.SetAttributeF64Request(
+            instrument=instr,
+            selector_string="",
+            attribute_id=nirfmxnr_types.NIRFMXNR_ATTRIBUTE_BANDWIDTH_PART_SUBCARRIER_SPACING,
+            attr_val=30e3,
+        )
+    )
+
+    if auto_level:
+        auto_level_response = client.AutoLevel(
+            nirfmxnr_types.AutoLevelRequest(
+                instrument=instr, selector_string="", measurement_interval=10.0e-3
+            )
+        )
+        reference_level = auto_level_response.reference_level
+        print(f"Reference level (dBm)        : {reference_level}\n")
+    else:
+        client.CfgReferenceLevel(
+            nirfmxnr_types.CfgReferenceLevelRequest(
+                instrument=instr, selector_string="", reference_level=0.0
+            )
+        )
+
+    client.SelectMeasurements(
+        nirfmxnr_types.SelectMeasurementsRequest(
+            instrument=instr,
+            selector_string="",
+            measurements=nirfmxnr_types.MEASUREMENT_TYPES_ACP
```

Comment:
I think you'll need to change this to `measurements_raw=` to appease mypy. It should work across the wire the same.